src/assets/wise5/classroomMonitor/dataExport/ComponentRevisionCounter.ts

```diff
@@ -0,0 +1,17 @@
+export class ComponentRevisionCounter extends Map<string, number> {
+  incrementCounter(nodeId: string, componentId: string): void {
+    const key = `${nodeId}_${componentId}`;
```

Comment:
Extract to func since it's duplicated in getCounter()?
```suggestion
    const key = this.getKey(nodeId, componentId);
```